binding/lua/docs/BENCHMARK.md

```diff
@@ -0,0 +1,44 @@
+#Multiverso Torch Binding Benchmark
+
+## Task Description
+
+Perform CIFAR-10 classification with torch resnet implementation.
+
+## Codebase
+
+[Microsoft/fb.resnet.torch multiverso branch](https://github.com/Microsoft/fb.resnet.torch/tree/multiverso)
+
+## Hardware
+
+- **Hosts** : 1
+- **GPU** : GeForce GTX TITAN X * 4
+- **CPU** : Intel(R) Core(TM) i7-5960X CPU @ 3.00GHz
+- **Memory** : 128GB
+
+## Common settings
+
+- batchSize 128
+- depth 32
+- nEpochs 164
+
+## Clarification for multiverso settings
+
+- Epoch means epoch for each worker with multiverso.
+- Learning rate is divided by the number of workers.
+- We choose master strategy to warm up the initial model.
+- Workers sync after each batch and has a barrier after each epoch.
+
+## Results
+
+| Code Name | #Process | #GPU | With multiverso | Best Model            |
+| :-------: | :------: | :--: | :-------------: | --------------------- |
+| 1P1G0M    | 1        | 1    | 0               | top1:7.634 top5:0.218 |
```

Comment:
Do you have any plugin to make nice tables in markdown files? I always want to find one...........